gradle.properties

```diff
@@ -22,3 +24,6 @@ bomProperty=micronautRedisVersion
 bomProperties=lettuceVersion
 
 testsredis=redis-lettuce/src/test/groovy/io/micronaut/configuration/lettuce
+
+org.gradle.caching=true
```

Comment:
I don't think we need this anymore in `gradle.properties`, right @melix ?

Reply:
In my tests, caching is not currently enabled by default

Reply:
I don't think we have this settings in other projects.

Reply:
No, I don't think we do...  I spotted it wasn't enabled whilst trying (unsuccessfully so far) to update the acme module, so I added it in here, as I assumed we wanted to enable caching.  @melix is that the plan?  Or should this be set by the settings plugin?

Reply:
it's used in several projects, and it's in the template project https://github.com/micronaut-projects/micronaut-project-template/blob/master/gradle.properties#L22

Reply:
what about the `org.gradle.parallel=true`.  I think we removed it from other projects because it caused issues with publication. And we use it with `--parallel` when running the tests in CI.

Reply:
parallel is enabled in several projects and doesn't interfere with publishing. If it does, I'd rather recommend to _only_ have the publishing task with `--no-parallel`, and have parallel enabled by default.